gateway/src/auth.ts

```diff
@@ -84,3 +91,6 @@ export async function changeProjectState(project: number, options: Pick<GatewayO
 
 const apiKeyCacheKey = (key: string, kvVersion: string) => `apiKeyAuth:${kvVersion}:${key}`
 const projectStateCacheKey = (project: number, kvVersion: string) => `projectState:${kvVersion}:${project}`
+
+const startsWithPaig = (headerValue: string | null): headerValue is 'Bearer paig' | 'paig' =>
```

Comment:
surely `headerValue is 'Bearer paig' | 'paig'` is wrong, that would mean it's literally one of those exact values?